ui/page/settings_page.go

```diff
@@ -89,8 +89,9 @@ func NewSettingsPage(l *load.Load) *SettingsPage {
 
 	pg.backButton, pg.infoButton = components.SubpageHeaderButtons(l)
 
-	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Theme, languagePreferenceKey,
-		values.DefaultLangauge, values.ArrLanguages).
+	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Theme,
+		languagePreferenceKey, values.DefaultLangauge, values.ArrLanguages,
+		pg.Icons.ContentClear).
```

Comment:
rather than use `pg.Theme`, lets kindly use `pg.Load`
```suggestion
	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Load,
		languagePreferenceKey, values.DefaultLangauge, values.ArrLanguages).
```